athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/domain/predicate/ConstraintEvaluator.java

```diff
@@ -81,7 +82,7 @@ public ConstraintEvaluator(BlockAllocator allocator, Schema schema, Constraints
      */
     public static ConstraintEvaluator emptyEvaluator()
     {
-        return new ConstraintEvaluator(null, SchemaBuilder.newBuilder().build(), new Constraints(new HashMap<>()));
+        return new ConstraintEvaluator(null, SchemaBuilder.newBuilder().build(), new Constraints(new HashMap<>(), Collections.emptyList(), Collections.emptyList(), -1));
```

Comment:
Lets make '-1' a static variable in the federation SDK.

Reply:
good idea, just pushed this change